app/allocation/controllers/view.js

```diff
@@ -44,6 +56,7 @@ module.exports = function view(req, res) {
       .map(move => ({
         id: move.id,
         person: move.person,
+        cancelAllocationHref: removeMoveLink(move),
```

Comment:
Oh boy - see my comment about differentiating the cancels and removes earlier. Is this for cancelling an allocation or removing a move?

Reply:
Ok -- I have renamed everything to use `remove move` 👍